app/templates/introduction.html

```diff
@@ -80,7 +80,11 @@ <h2 class="neptune u-mb-xs" data-qa="legal-response">{{ _("Your response is lega
 {% endblock %}
 {% block footer %}
 <div class="container u-mt-xl u-pt-l">
-    {% include theme('partials/feedback/expandable_inline.html') %}
+  <div class="grid">
+    <div class="grid__col col-7@m">
+      {% include theme('partials/feedback/expandable_inline.html') %}
```

Comment:
All usage of `explandable_inline.html` is wrapped in:
```
<div class="container u-mt-xl u-pt-l">
  	<div class="grid">
    	<div class="grid__col col-7@m">
```
So is it worth just adding these to `expandable_inline.html` and reduce duplication?